ml-agents/tests/trainers/test_barracuda_converter.py

```diff
@@ -0,0 +1,23 @@
+import unittest.mock as mock
+import pytest
+import os
+import tempfile
+
+import mlagents.trainers.tensorflow_to_barracuda as tf2bc
+
+
+def test_barracuda_converter():
+    path_prefix = os.path.dirname(os.path.abspath(__file__))
+    tmpfile = os.path.join(tempfile._get_default_tempdir(), next(tempfile._get_candidate_names()) + '.nn')
+
+    # make sure file doesn't exist before conversion
+    assert (not os.path.isfile(tmpfile))
```

Comment:
This could break if the line 23 is not called. (Because of errors in the asset calls). This test would fail thereafter. I would remove the file if exists at the beginning of the test. Also, Some comments on this method.

Reply:
will fix it

Reply:
fixed